renderer/hocs/withInputValidation.js

```diff
@@ -23,15 +25,22 @@ const withRequiredValidation = Component =>
     }
 
     validate = (value = '') => {
-      const { isDisabled, isRequired } = this.props
+      const { isDisabled, isRequired, minLength, maxLength } = this.props
       if (isDisabled) {
         return
       }
       try {
+        let validator = yup.string()
         if (isRequired) {
-          const validator = yup.string().required()
-          validator.validateSync(value)
+          validator = validator.required()
         }
+        if (minLength) {
+          validator = validator.min(minLength)
+        }
+        if (maxLength) {
+          validator = validator.max(maxLength)
+        }
```

Comment:
Will this validate correctly BOTH `maxLength` and `minLength` at a time?

Reply:
It should do - each conditional adds an additional check to the validator